src/index.css

```diff
@@ -1,13 +1,21 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+  padding: 0;
+}
+
+body {
+  min-height: 100vh;
+  background: linear-gradient(#4988b2e8, #b8c9e6);
+  font-family: fantasy;
```

Comment:
I'm not sure where this "fantasy" font-family is coming from. The stylesheet you imported in the `index.html` uses a "Rubik Burned" font-family name. You can verify this by visiting the [font stylesheet](https://fonts.googleapis.com/css2?family=Rubik+Burned&display=swap) in your browser and look for the `font-family` property.

On a side note, you'll always want to provide fallback options (like you did previously -- now commented out). Otherwise, if this font fails to load (for some reason), then the fallback will be the user's system default which can break your website's layout.